daffodil-core/src/main/scala/org/apache/daffodil/dsom/ElementBase.scala

```diff
@@ -288,7 +291,10 @@ trait ElementBase
     }
   }
 
-  private lazy val parentMinimizedScope = enclosingElement.map { _.minimizedScope }.getOrElse(scala.xml.TopScope)
+  private lazy val parentMinimizedScope = {
+    val ee = enclosingElements.headOption // FIXME: doesn't work unless all are same.
```

Comment:
Ah. I have not dealt with the minimized scope issue as yet. That remains a TODO/FIXME issue here. 
This is assuming the right answer for parentMinimizedScope is that of the first shared instance. Because we only share things that share properties (shared PropEnv), this works in most cases. Not clear it will for this scope minimizaiton stuff because that's not really about properties.  So this will need fixing.

Reply:
Gotcha. Not worth fixing as part of this cleanup. Maybe just include the bug number so it's clear what this fixup references?